### PR TITLE
integration: watch test for multi-events with txn

### PR DESCRIPTION
**Pull request by gyuho**
Related to https://github.com/coreos/etcd/issues/4216.

### CODE REVIEW

File: integration/v3_grpc_test.go [hunk 1]
```diff
@@ -560,6 +559,76 @@ func TestV3WatchMultiple(t *testing.T) {
...
+	events := []*storagepb.Event{}
+	for len(events) != 3 {
+		resp, err := wStream.Recv()
+		if err != nil {
+			t.Errorf("wStream.Recv error: %v", err)
+		}
+		if resp.Created {
+			continue
+		}
+		events = resp.Events
```
Comment (heyitsanthony) [inline integration/v3_grpc_test.go]: `append(events, resp.Events...)`?

File: integration/v3_grpc_test.go [hunk 2]
```diff
@@ -560,6 +559,76 @@ func TestV3WatchMultiple(t *testing.T) {
...
+	tresp, err := kvc.Txn(context.Background(), &txn)
+	if err != nil {
+		t.Fatalf("kvc.Txn error: %v", err)
+	}
+	if !tresp.Succeeded {
+		t.Fatalf("kvc.Txn failed: %+v", tresp)
+	}
+
+	events := []*storagepb.Event{}
+	for len(events) != 3 {
```
Comment (heyitsanthony) [inline integration/v3_grpc_test.go]: `len(events) < 3`

File: integration/v3_grpc_test.go (hunk 1 above)
Comment (gyuho) [inline integration/v3_grpc_test.go]: Thanks will fix. And forgot to sort before comparison.

**Comment by gyuho** [comment]
@heyitsanthony Just fixed. PTAL. Thanks,

File: integration/v3_grpc_test.go [hunk 3]
```diff
@@ -560,6 +563,84 @@ func TestV3WatchMultiple(t *testing.T) {
...
+	rok, nr := WaitResponse(wStream, 1*time.Second)
+	if !rok {
+		t.Errorf("unexpected pb.WatchResponse is received %+v", nr)
+	}
+
+	// can't defer because tcp ports will be in use
+	clus.Terminate(t)
+}
+
+type storagepbEvents []*storagepb.Event
```
Comment (heyitsanthony) [inline integration/v3_grpc_test.go]: `eventsSortByKey`

**Comment by heyitsanthony** [comment]
lgtm after fixing nits

**Comment by gyuho** [comment]
Thanks. Just fixed. Will merge after CI passes.

**Pull request merged**